README.rst

```diff
@@ -18,9 +18,65 @@ Results are currently hosted in an ad hoc manner, you may find some results `her
 <http://hera.physchem.kth.se/~sympy_asv>`__ and `here <http://www.moorepants.info/misc/sympy-asv/>`__.
 
 The benchmarks are run using `airspeed velocity
-<http://spacetelescope.github.io/asv>`__.
+<https://asv.readthedocs.io/>`__.
 
 When adding benchmarks, mirror the SymPy directory tree for the feature you are
 adding. If your benchmark is not for a specific feature such that creating
 subdirectories for your benchmark is not reasonable, put your benchmark in the
 top level benchmarking directory.
+
+
+Quickstart
+----------
+
+Installation
+^^^^^^^^^^^^
+
+Firstly, you need to install ``asv``
+
+``pip install asv``
+
+or in conda environment, use
+
+``conda install -c conda-forge asv``
+
+And then you may also need to install ``virtualenv``. Because we have
+``"environment_type": "virtualenv"`` in ``asf.conf.json``.
+
+``pip install virtualenv``
+
+or in conda environment, use
+
+``conda install -c conda-forge virtualenv`` to install the dependency.
```

Comment:
I can't imagine that this is a good idea.

i would be better to recommend using conda environments in asf.conf.json.

Reply:
probably, I can't remember why we opted to keep `virtualenv` as the default.

Reply:
I just swap out the conf file on my benchmark machine and use conda. Here is an issue: https://github.com/sympy/sympy_benchmarks/issues/4

Reply:
And there was this attempt: https://github.com/sympy/sympy_benchmarks/pull/42

Reply:
Ah, now I remember. @sylee957 opened https://github.com/sympy/sympy_benchmarks/pull/54, I added the auto-selection logic to `run_benchmarks.sh`